src/pages/Lend/LoanChannelCategoryPage.vue

```diff
@@ -289,6 +297,27 @@ export default {
 			if (!_isEqual(this.$route.query, this.urlParams)) {
 				this.$router.push({ query: this.urlParams });
 			}
+		},
+		getAlgoliaFilterUrl() {
```

Comment:
This method builds the algolia compatible facet params if we have them in the new map file.